android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -504,6 +515,33 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 		}
 	}
 
+	private void zoomBy(WebView webView, float scale)
+	{
+		if (webView != null) {
+			if (scale < 0.0f) {
+				scale = 0.0f;
```

Comment:
The minimum scale needs to be floored to `0.1f`. A value less than this will cause an exception to be thrown.